src/test/java/LoginTests.java

```diff
@@ -3,23 +3,33 @@
 import org.openqa.selenium.chrome.ChromeOptions;
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LogInPage;
 
 import java.time.Duration;
 
+//login with valid email using object method
 public class LoginTests extends BaseTest {
     @Test
-    public void loginEmptyEmailPassword() {
+    public void loginValidEmailPasswordTest() {
+        //creating objects
+        LogInPage loginPage = new LogInPage(driver);
+        HomePage homePage = new HomePage(driver);
+        LogInPage.provideEmail("aida.taymaskhanova@testpro.io");
+        LogInPage.providePassword("Ozzikpozzik18");
+        LogInPage.clickSubmit();
+        Assert.assertTrue(homePage.getUserAvatar().isDisplayed());
 
 //      Added ChromeOptions argument below to fix websocket error
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--remote-allow-origins=*");
+        //ChromeOptions options = new ChromeOptions();
+        //options.addArguments("--remote-allow-origins=*");
 
-        WebDriver driver = new ChromeDriver(options);
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        //WebDriver driver = new ChromeDriver(options);
+        //driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
 
-        String url = "https://qa.koel.app/";
-        driver.get(url);
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-        driver.quit();
+        //String url = "https://qa.koel.app/";
+        //driver.get(url);
+        //Assert.assertEquals(driver.getCurrentUrl(), url);
+        //driver.quit();
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.